components/sensor/pulse_counter.rst

```diff
@@ -73,6 +77,31 @@ count the light pulses on a power meter, you can do the following:
         filters:
           - multiply: 0.06
 
+
+Counting total pulses
+---------------------
+
+When the total sensor is configured, the pulse_counter also reports the total 
+number of pulses measured. When used on a power meter, this can be used to 
+measure the total consumed energy in kWh. 
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    sensor:
+    - platform: pulse_counter
+        pin: 12
+        unit_of_measurement: 'kW'
+        name: 'Power Meter House'
+        filters:
+          - multiply: 0.06
+
+        total:
+          unit_of_measurement: 'kWh'
+          name: 'Energy Meter House'
+          filters:
+            - multiply: 0.001
```

Comment:
@FrankBakkerNl if the counter is outputing in kW and the total in kWh, why do you need to multiply by 0.001 here?

Reply:
The total counter counts the actual pulses, not what is output by the kW sensor. The kW sensor also has a multiply 0.06 here. I do not remember the math 100% but I think that translates from minutes to hours and W to kW